ci-operator/config/openshift/release/openshift-release-master__origin-4.4-nightly-to-4.4-nightly.yaml

```diff
@@ -0,0 +1,33 @@
+base_images:
+  base:
+    name: "4.4"
+    namespace: ocp
+    tag: base
```

Comment:
Huh are these actually needed?

Reply:
I can check, but IIRC a lot of steps still use `base` as their image (even though it's usually unnecessary and should set to a static `centos:8` or similar) so it needs to be set as a `base_image`.